docs/spec_changelog.md

```diff
@@ -0,0 +1,43 @@
+---
+id: "spec_changelog"
+title: "Twirp Wire Protocol Changelog"
+sidebar_label: "Changelog"
+---
+
+This document lists changes to the Twirp specification.
+
+## Changed in v6
+
+### URL scheme
+
+In [v5](./PROTOCOL.md), URLs followed this format:
+
+```bnf
+**URL ::= Base-URL "/twirp/" [ Package "." ] Service "/" Method**
+```
+
+Version 6 changes this format to remove the mandatory `"/twirp"` prefix, and
+changes `Package` and `Service` to be delimited with a `/` instead of a `.`
+character:
+
+```bnf
+**URL ::= Base-URL "/" [ Package "/" ] Service "/" Method**
+```
```

Comment:
I'm not sure if there was already a discussion about using "." vs "/" for the package separator, but I am definitely not convinced that doing the `. -> /` change is a good thing. Let me list the advantages that I can see with each one:

Advantages of "/" separator:
  * Looks nicer when described like this: `Base/Package/Service/Method`.
  * It is different from gRPC routes. This was mentioned before as a pro, but I don't clearly see the advantage of this; only the very trained eye will be able to tell the difference at a glance, and I don't see the advantage of keeping a Twirp and a gRPC service both in the same Base-URL, only differentiated by the "." vs "/" syntax.
  * If there is no package or the package has no dots, and the Base-URL also has no dots, then the resulting URL will not have dots. Maybe this is easier to recognize by some middleware?

Advantages of "." separator:
  * Somewhat backwards-compatible. With the "/", service implementations will need to explicitly handle both new and old paths. With the "." this is not entirely true, because even if the service didn't support old routes, the user could still add the "/twirp" prefix on the Base-URL to deal with compatibility.
  * Since packages are optional, the "." makes it easier to tell if a URL has a package (`blahblah.com/api/myserviceapi.CoolAPI/CreateStuff`) or not (`blahblah.com/api/CoolAPI/CreateStuff`). 
 * Avoids service path conflicts. For example, given the services `Foo1` with package `package` and name `Foo`, and `Foo2` with no package and the same name `Foo`. With the "." a URL like `http://fooservice.com/package/Foo/Method` is clearly mapping to `Foo2` with the Base-URL `http://fooservice.com/package`, but with the "/" the same URL is ambiguous. This is probably an edge case because in most cases the user controls the Base-URL, but it is still possible.

I may be missing some other point (please comment!), but the advantages of the "/" separator look a little weak to me.

Reply:
Excellent points, and I agree that we under-designed this. I think PR feedback is a tough place to do design discussion, though. Could you chime in on #86? You can just paste in what you have here :)

Reply:
Ok I'll add this points to #86. Now I realize they are very similar to what @rhysh was talking about, but presented in a different order. It may help to add clarity to design decisions.